ConwaysGameOfLife.Tests/CellShould.cs

```diff
@@ -54,5 +54,15 @@ public void WhenCellStateAndLocationAreSame_BeEqual()
             Assert.Equal(expected, result);
         }
 
+        [Fact]
+        public void WhenUniverseRegenerated_SwitchState()
+        {
+            var cell = new Cell(CellState.Dead, 0, 0);
+            var expected = new Cell(CellState.Alive, 0, 0);
+
+            var result = cell.SwitchCellState(cell);
+
+            Assert.Equal(expected, result);
+        }

```

Comment:
Do we want the Cell to know how to regenerate itself? Or is it "dumb" and some higher class is responsible for regenerating the Universe as a whole, and thereby the Cells?